llumnix/global_scheduler/migration_scheduler.py

```diff
@@ -31,12 +31,18 @@ class PairMigrationConstraints(str, Enum):
     DECODING_2_DECODING = "DECODING_2_DECODING"
     PREFILL_2_DECODING = "PREFILL_2_DECODING"
 
+class MigrationFilterConfig:
+    def __init__(self, migrate_out_load_threshold):
+        self.migrate_out_load_threshold: float = migrate_out_load_threshold
+
 class MigrationScheduler:
     def __init__(self,
                  pair_migration_policy: str,
                  migrate_out_load_threshold: float,
                  instance_load_calculator: InstanceLoadCalculator) -> None:
-        self.migrate_out_load_threshold = migrate_out_load_threshold
+        self.filter_config = MigrationFilterConfig(migrate_out_load_threshold=migrate_out_load_threshold)
+        self.migration_filter = MigrationFilter(self.filter_config)
```

Comment:
The naming of `MigrationFilterConfig` and `MigrationFilter` look confusing. Additionally, self.filter_config seems to be of no use.

Reply:
MigrationFilterConfig is the configuration class for MigrationFilter. It has been introduced to enhance extensibility, so we should just keep it as is.